x-pack/plugins/fleet/server/types/rest_spec/agent_policy.ts

```diff
@@ -20,33 +20,42 @@ import { validateKuery } from '../../routes/utils/filter_utils';
 import { BulkRequestBodySchema } from './common';
 
 export const GetAgentPoliciesRequestSchema = {
-  query: schema.object({
-    page: schema.maybe(schema.number({ defaultValue: 1 })),
-    perPage: schema.maybe(schema.number({ defaultValue: 20 })),
-    sortField: schema.maybe(schema.string()),
-    sortOrder: schema.maybe(schema.oneOf([schema.literal('desc'), schema.literal('asc')])),
-    showUpgradeable: schema.maybe(schema.boolean()),
-    kuery: schema.maybe(
-      schema.string({
-        validate: (value: string) => {
-          const validationObj = validateKuery(
-            value,
-            [LEGACY_AGENT_POLICY_SAVED_OBJECT_TYPE],
-            AGENT_POLICY_MAPPINGS,
-            true
-          );
-          if (validationObj?.error) {
-            return validationObj?.error;
-          }
-        },
-      })
-    ),
-    noAgentCount: schema.maybe(schema.boolean()),
-    full: schema.maybe(schema.boolean()),
-    format: schema.maybe(
-      schema.oneOf([schema.literal(inputsFormat.Simplified), schema.literal(inputsFormat.Legacy)])
-    ),
-  }),
+  query: schema.object(
+    {
+      page: schema.maybe(schema.number({ defaultValue: 1 })),
+      perPage: schema.maybe(schema.number({ defaultValue: 20 })),
+      sortField: schema.maybe(schema.string()),
+      sortOrder: schema.maybe(schema.oneOf([schema.literal('desc'), schema.literal('asc')])),
+      showUpgradeable: schema.maybe(schema.boolean()),
+      kuery: schema.maybe(
+        schema.string({
+          validate: (value: string) => {
+            const validationObj = validateKuery(
+              value,
+              [LEGACY_AGENT_POLICY_SAVED_OBJECT_TYPE],
+              AGENT_POLICY_MAPPINGS,
+              true
+            );
+            if (validationObj?.error) {
+              return validationObj?.error;
+            }
+          },
+        })
+      ),
+      noAgentCount: schema.maybe(schema.boolean()),
+      full: schema.maybe(schema.boolean()),
+      format: schema.maybe(
+        schema.oneOf([schema.literal(inputsFormat.Simplified), schema.literal(inputsFormat.Legacy)])
+      ),
+    },
+    {
+      validate: (query) => {
```

Comment:
Do we call the API anywhere from the UI where this could cause an issue?

Reply:
I just did a more in depth check and I fixed the few places where we were still calling that API without noAgentCount